### PR TITLE
Added benchmarks for thea-council and xcm-helper

### PR DESCRIPTION
**Pull request by zktony**
Added benchmarks for thea-council and xcm-helper.

### CODE REVIEW

**Review by 35359595 — CHANGES_REQUESTED** [review]
please merge #26 and make all benchmarks running

File: pallets/xcm-helper/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	withdraw_asset {
+		let b in 1 .. 1000;
+		let asset_location = MultiLocation::new(1, Junctions::X1(Junction::Parachain(0)));
+		let asset_id = AssetId::Concrete(asset_location);
+		let multi_asset = MultiAsset::from((asset_id, Fungibility::Fungible(1000000000)));
+		let wrapped_multi_asset = VersionedMultiAssets::V1(MultiAssets::from(multi_asset));
+		let boxed_asset_id = Box::new(wrapped_multi_asset);
+		let destination = MultiLocation::new(1, Junctions::X2(Junction::Parachain(2011), Junction::AccountId32 { network: NetworkId::Any, id: [1;32] }));
+		let boxed_multilocation = Box::new(VersionedMultiLocation::V1(destination));
+		let boundec_vec:  BoundedVec<
```
Comment (35359595) [inline pallets/xcm-helper/src/benchmarking.rs:58]: max size of this vector is 10 so populate it to max capacity for worst case benchmarking.

**Review by 35359595 — CHANGES_REQUESTED** [review]

File: pallets/xcm-helper/src/benchmarking.rs
Comment (serhii-temchenko) [inline pallets/xcm-helper/src/benchmarking.rs:1]: `change_thea_key` extrinsic is not covered. Please add.

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

**Comment by Gauthamastro** [comment]
Stale

**Pull request closed without merging**